pkg/operator/starter.go

```diff
@@ -148,6 +148,10 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 
 	configUpgradeableController := unsupportedconfigoverridescontroller.NewUnsupportedConfigOverridesController(operatorClient, ctx.EventRecorder)
 
+	if ctx.Server != nil {
+		ctx.Server.Handler.NonGoRestfulMux.Handle("/debug/controllers/resourcesync", debugHandler)
```

Comment:
I suspect that we'll want `handleprefix` in the end, but ok.

Reply:
Can ctx.Server be nil?